tests-web/www/js/main.js

```diff
@@ -146,7 +138,7 @@ class TestCase {
 
       importButton.innerText = "Importing OSM data...";
 
-      const drivingSide = app.importSettings.drivingSideForNewImports;
+      const drivingSide = app.getImportSettings().drivingSideForNewImports || "Right";
```

Comment:
Ideally, the choice of driving side would be presented alongside the "import" button that uses it, instead of it being part of "import settings".